review.md

```diff
@@ -2,48 +2,73 @@
-
-## The manual review process
-
-If a pull request is flagged for manual review, an R-multiverse moderator will read the pull request and ask questions if necessary. Although the moderator may make optional suggestions on a case-by-case basis, package reviews must be consistent, reliable, and inclusive whenever possible. The decision to close or merge the pull request must be based exclusively on the following pre-defined list of requirements:
-
-1. Each contribution must comply with the [code of conduct](conduct.md). Examples of prohibited content include profanity, malicious behavior, security risks, copyright violations, and other conduct which could reasonably be considered inappropriate in a professional setting. All this applies to the package, the URL, any other metadata in the contribution, and the contents of the package itself. 
-1. The package name, URL, and all other metadata must be complete and correct.
-1. Each text file must apply to only one package.
-1. The text file name must be the name of the package.
-1. For JSON listings, the `"branch"` field must be `"*release"` (except in specific predetermined cases such as packages in <https://github.com/cran>), the `"subdirectory"` field must be supplied and exist, the `"url"` field must exist and be correct, and the `"package"` field must agree with the name of the text file.
-1. Each contributed URL must point to an existing GitHub or GitLab repository.
-1. The URL must be the true/official location of the source code or a faithful mirror of the true location. The package maintainers have the authority to choose the URL. Unofficial or unsupported forks should not be included. The moderator must use discretion on a case-by-case basis because sometimes a fork becomes the true version (e.g. if the original maintainer abandons a package and becomes unreachable indefinitely).
-1. The URL must have a release on GitHub or GitLab so R-universe can process the package without error. As a last resort, if the maintainer does not provide their own releases, a repository from the CRAN mirror at <https://github.com/cran> may be registered.
-1. If the listing of an existing package is modified, then the moderator must verify that the new information is complete and correct. In many cases, it may be necessary to obtain permission from the package maintainer.
-1. The reasons for deleting a package listing may vary on a case-by-case basis. The moderator must carefully consider the impact that deletion would have on the community and on reverse dependencies. In many cases, it may be necessary to obtain permission from the package maintainer.
+This policy dictates the process by which R-multiverse reviews and accepts contributed R packages.
+All contributions must comply with R-multiverse policies, including but not limited to [Acceptable Use](https://r-multiverse.org/aup.html), [Terms of Use](https://r-multiverse.org/terms.html), and [Code of Conduct](https://r-multiverse.org/conduct.html).
```

Comment:
Use relative links rather than absolute links, or Quarto document links like @shikokuchuo does? https://github.com/r-multiverse/r-multiverse.github.io/blob/6e3040bddda216739ee7098fa4caec3c3186ceb2/contributors.md?plain=1#L74

Reply:
Good idea, changed in https://github.com/wlandau/r-multiverse.github.io/commit/fc8e265b9025eb21eaf5fba629916465b8815ad4.